sandy/core/endf6.py

```diff
@@ -2475,3 +2475,162 @@ def covariance_info(self, nubar=True, xs=True, mubar=True, chi=True):
             'chi': run_chi if chi else False,
             }
         return cov_info
+
+    def _get_xs_cov(self, mf=[31, 33], mt=range(1, 10000),
```

Comment:
Why do we need a mf? Thus could be by default 33. And then have another method `get_nubar_xs` with 31 as default. In theory these two methods should both call a common third method